rabbit-image/README.md

```diff
@@ -0,0 +1,188 @@
+# RabbitMQ Cluster Image
```

Comment:
I wrote down the 50-thousand-foot idea behind this image and provided an example yaml for hands-off deployment.